### DISCUSSION TITLE
No me sale Dune basics

### DISCUSSION THREAD

**Discussion by Qubemid3** [cpp-review-dune/introductory-review#18] [Q&A] [answered]
¿Qué tal Carlos? Sabes, no puedo concretar más allá de los códigos de `$ man dunecontrol`, aquí está mi [grabación](https://asciinema.org/a/QimbKXKGGnMP7d36mXZz4IJMa).

Porfa ayúdame con este tema, no sé si será porque en la mañana subí al GitHub el ejemplo que hice y eso esta haciendo conflicto con lo que estoy practicando.

**Comment by carlosal1015** [comment]
Hola @Qubemid3, no veo ningún problema serio. Me llamó la atención que no reconocería el manual, quizás deba corregirlo en el PKGBUILD. Le voy a echar un vistazo ahora.

**Reply by Qubemid3** [reply]
Ya ps sino lo vemos mañana mejor quiza te tome más tiempo.

**Reply by carlosal1015** [reply]
Bien, felizmente, [no es el PKGBUILD](https://asciinema.org/a/472049), más debo ver el dockerfile.

**Comment by carlosal1015** [comment]
Ya encontré la [respuesta](https://bbs.archlinux.org/viewtopic.php?id=43689), cuando se actualizó se creó el archivo `pacmanew` y se añadió unas líneas finales al archivo `/etc/profile`:
![image](https://user-images.githubusercontent.com/21283014/155866808-446e3716-7450-4d49-8252-6f69d91d722e.png)

```
# Man is much better than us at figuring this out
unset MANPATH
```

**Comment by carlosal1015** [comment, marked as answer]
Ahora, sí, lo que pasaba es que [el archivo `/etc/pacman.conf` tenia la opción de no instalar las páginas del manual](https://asciinema.org/a/472052) de cualquier paquete nuevo que se instale.

![image](https://user-images.githubusercontent.com/21283014/155867197-08585e2e-09ac-424c-b417-059e8b2dc24b.png)


**Solución** De esta forma [se solucionaría](https://asciinema.org/a/472054), editaré el dockerfile para que no venga con esa opción.
